test/unit/lib/mapLink.js

```diff
@@ -85,6 +85,57 @@ describe('mapLink', () => {
       expect(results[0].mapUrl).to.be.equal(expectedMapLink);
     });
 
+    it('should leave start address empty for place searches', () => {
+      const yourLocationSearchCriteria = {
+        coordinates,
+        searchTerm,
+        searchType: constants.placeSearch,
+      };
+      const inputList = [{
+        name: 'name',
+        address: {
+          line1: 'line1',
+        },
+      }];
+      const destination = 'name,line1';
+      const params = {
+        saddr: '',
+        daddr: `${destination}`,
```

Comment:
Could this just be `daddr: destination` rather than interpolating the variable?
Same goes for the following line and the two reference in the test below.

Reply:
Didn't register that it had been written like that, I've corrected them now.